### ISSUE TITLE
Make the Grief Report crop actually crop pictures when in picture-taking mode

### ISSUE THREAD

**Issue by Arcadius2006** [LittleBigRefresh/Refresh#457]
pretty self explanatory - the pictures taken using grief reports should be cropped to the dimensions of the highlight box instead of the entire image being sent as-is

**Comment by jvyden** [comment]
I don't know if we actually want to do this one. I think it's probably better to show the orange border when hovering over the image similar to how it's presented while you're highlighting in-game.

**Comment by Arcadius2006** [comment]
while that works for standard grief report stuff, the idea is to only do it in "send as picture" mode so you dont have to go and crop the image in another program

**Comment by jvyden** [comment]
Yes, we're on the same page. I only see it making things harder on the frontend, since that introduces even more non-16:9 aspect ratios. We're trying to move away from masonry grids and this works against that.

I recommend [ShareX](https://getsharex.com/) for easier cropping.

**Comment by jvyden** [comment]
@Beyley thoughts?

**Comment by Beyley** [comment]
i'd personally prefer we didn't crop, and instead just showed the outline of the selected area on the site

the game does not say that the area you selected is a "crop" anyway, its just asking "what part of the image has the offending content", so i feel its fine to interpret that as a highlight box and not a crop

**Comment by Arcadius2006** [comment]
what if on the website it highlighted the image, but if you clicked the highlight, it brought you to a cropped version of it?

**Comment by jvyden** [comment]
Could be a fair middle-ground... just having a `Show cropped` button or something.

Would have to be a client-side (as in refresh-web) feature though.

**Comment by Beyley** [comment]
Closing in favour of #559
